### DISCUSSION TITLE
Windows maps

### DISCUSSION THREAD

**Discussion by AlanAnderson-BlueScope** [CommunityToolkit/Maui#1274] [General]
I'm having an issue that the windows maps implementation doesn't switch to the Satellite view. Street and Hybrid seem to work as expected, but Satellite is the same as Hybrid. Has anyone else had this issue? It works fine on iOS and Android with the same code. 

Thanks
Alan
<img width="1476" alt="Satellite view" src="https://github.com/CommunityToolkit/Maui/assets/137911895/9cbef167-8bdd-45a6-9312-ca4d9cc2e662">

**Comment by VladislavAntonyuk** [comment]
Check if windows maps support this view.

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
